__tests__/01_basic_spec.ts

```diff
@@ -98,6 +98,39 @@ describe('bind', () => {
     expect(p?.foo?.bar).toBe('b');
     expect(m.get('foo').get('bar')).toBe('b');
   });
+
+  it('is a single transaction', async () => {
```

Comment:
As per some of the discussions in the issues, this should also allow unsubscribing and bind being a single transaction.